.travis.yml

```diff
@@ -50,7 +50,7 @@ env:
     # - INSTANCE: default-amazonlinux-2-2018-3-py2
     # - INSTANCE: default-debian-8-2017-7-py2
     # - INSTANCE: default-ubuntu-1604-2017-7-py2
-    - INSTANCE: default-centos-6-2017-7-py2
+    - INSTANCE: centos6-centos-6-2017-7-py2
```

Comment:
Isn't this a typo?

Reply:
@baby-gnu No, this is correct in the current state of this PR (i.e. introducing a `centos6` suite).  The structure is:

```yaml
    - INSTANCE: {{ suite_name }}-{{ platform }}
```

And due to the [`includes`](https://github.com/saltstack-formulas/template-formula/pull/159/files#diff-fdbca9ed07a634829a3548f874dc925aR157) and [`excludes`](https://github.com/saltstack-formulas/template-formula/pull/159/files#diff-fdbca9ed07a634829a3548f874dc925aR137), this is what is available via. `kitchen list -b`:

```bash
template-formula$ kitchen list -b
default-debian-10-develop-py3
default-ubuntu-1804-develop-py3
default-centos-7-develop-py3
default-fedora-30-develop-py3
default-opensuse-leap-15-develop-py3
default-amazonlinux-2-develop-py2
default-debian-9-2019-2-py3
default-ubuntu-1804-2019-2-py3
default-centos-7-2019-2-py3
default-fedora-30-2019-2-py3
default-opensuse-leap-15-2019-2-py3
default-amazonlinux-2-2019-2-py2
default-debian-9-2018-3-py2
default-ubuntu-1604-2018-3-py2
default-centos-7-2018-3-py2
default-fedora-29-2018-3-py2
default-opensuse-leap-15-2018-3-py2
default-amazonlinux-2-2018-3-py2
default-debian-8-2017-7-py2
default-ubuntu-1604-2017-7-py2
default-fedora-29-2017-7-py2
default-opensuse-leap-15-2017-7-py2
default-amazonlinux-2-2017-7-py2
centos6-centos-6-2017-7-py2
```